### PR TITLE
[5.x] Organize user controllers

### PR DESCRIPTION
**Pull request by jasonvarga**
Reviewing #8924 would have a fair bit simpler if the various tags/actions had their own controllers.

This PR does that, plus a little more general cleanup.

### CODE REVIEW

File: src/Http/Controllers/User/PasswordController.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{
+    public function __invoke(UserPasswordRequest $request)
+    {
+        $user = User::current();
+
+        $user->password($request->password);
+
+        $user->save();
+
+        return $this->successulResponse();
```
Comment (edalzell) [inline src/Http/Controllers/User/PasswordController.php:18]: typo `successFulResponse`

Comment (jasonvarga) [reply src/Http/Controllers/User/PasswordController.php:18]: Thanks. It even told me. Whoops.

CleanShot 2024-05-16 at 09 51 24

**Comment by chatelardfrancois** [comment]
When is this change going to be released ? The invokable controllers change is much needed

**Comment by duncanmcclean** [comment]
This change was released earlier this year, in [v5.3.0](https://github.com/statamic/cms/releases/tag/v5.3.0).

**Pull request merged**